docs/styleguide/PyGuide.md

```diff
@@ -264,263 +239,54 @@ class C(ABC):
     @abstractmethod
     def my_abstract_method(self, ...):
         pass
-    
+
     @classmethod
     @abstractmethod
     def my_abstract_classmethod(cls, ...):
         pass
-    
+
     @staticmethod
     @abstractmethod
     def my_abstract_staticmethod(...):
         pass
-    
+
     @property
     @abstractmethod
     def my_abstract_property(self):
         pass
-    
+
     @my_abstract_property.setter
     @abstractmethod
     def my_abstract_property(self, val):
         pass
-    
+
     @abstractmethod
     def _get_x(self):
         pass
-    
+
     @abstractmethod
     def _set_x(self, val):
         pass
 ```
 
 
-<a id="s3-python-style-rules"></a>
-<a id="3-python-style-rules"></a>
+<a id="s4-python-style-rules"></a>
+<a id="4-python-style-rules"></a>
 <a id="python-style-rules"></a>
-## 3 Python Style Rules 
-
-<a id="s3.1-line-length"></a>
-<a id="31-line-length"></a>
-<a id="line-length"></a>
-### 3.1 Line length 
-
-Maximum line length is *120 characters*.
-
-Explicit exceptions to the 120 character limit:
-
--   Long import statements.
```

Comment:
It looks like this list is a list of exceptions when we can disable auto-formatting.

Reply:
I assume that isort will only be disabled for cases where autoformatting breaks something like in the case of `konia`.

I will add cases when disabling the formatter is acceptable.